### ISSUE TITLE
Inference on single image\folder

### ISSUE THREAD

**Issue by iariav** [shreyaskamathkm/FTNet#2]
Hi,
thanks for the great work.
I want to test some of the pre-trained models on my own data.
is there some simple way to run a pre-trained model on a folder of images? or on a single image?
thanks

**Comment by Harsh188** [comment]
This would be quite useful.

**Comment by shreyaskamathkm** [comment]
Currently we are in the process of updating the repo with the latest packages. We should have this functionality soon.

**Comment by Harsh188** [comment]
Is there a quick solution to running inference using fine-tuned ckpt? I want to visualize results on thermal data that I've captured.

**Comment by shreyaskamathkm** [comment]
I have something on the 'develop' branch. But it's work in progress. Should be able to publish it once I refactor it.

**Comment by shreyaskamathkm** [comment]
The  [develop](https://github.com/shreyaskamathkm/FTNet/tree/develop) branch has been updated to support inferring on a folder of images. Please use that branch and follow the instructions. Please ensure you have the downloaded  `rev-2` models from the box. The old model checkpoints will not work.

**Comment by kulkarnikeerti** [comment]
@shreyaskamathkm I am using [develop](https://github.com/shreyaskamathkm/FTNet/tree/develop) branch to infer a set of images. But, I also wanted to try training with the same branch as I have lot issues with the main branch due to a lot of packages incompatibilities. I ran the mentioned command `python -m ftnet -c ./ftnet/cfg/train_soda.toml` for the training and it runs just for one epoch although the mentioned number of epochs is 100. I tried to debug and find out if somewhere in the code 1 was passed in epochs but I could not find. Could you please help me if that actually is the case in this branch?

Thanks in advance

**Comment by shreyaskamathkm** [comment]
@kulkarnikeerti  Yes, I think I forgot to set debug = False in the config file. Can you try setting debug=False and rerun that? 

When debug is True, the setup makes sure to enable fast_dev which is a PyTorch Lightning flag to check if everything is working and run for 1 epoch

**Comment by kulkarnikeerti** [comment]
Yes, that works. Thank you!
